projects/packages/forms/src/blocks/form-progress-indicator/index.js

```diff
@@ -57,8 +57,40 @@ export const settings = {
 	edit: edit,
 	save: save,
 	attributes: {},
+	styles: [
+		{
+			name: 'line',
+			label: __( 'Line', 'jetpack-forms' ),
+			isDefault: true,
+		},
+		{
+			name: 'dots',
+			label: __( 'Dots', 'jetpack-forms' ),
+		},
+	],
 	transforms: {},
 	example: {},
+	deprecated: [
+		// Version with old jetpack-form-progress-indicator-bar structure
+		{
+			apiVersion: 2,
```

Comment:
Confuses me that this was needed.

Reply:
I tried several versions of the `save()` method on v3, but none matched. Not sure what the issue is, I’d love to understand it better.